test/ffmpeg-qsv/encode/hevc.py

```diff
@@ -89,7 +89,8 @@ def test_r2r(self, case, gop, slices, qp, quality, profile):
     self.encode()
 
 class cbr(HEVC8EncoderTest):
-  def init(self, tspec, case, gop, slices, bframes, bitrate, fps, profile):
+  def init(self, tspec, case, gop, slices, bframes, bitrate, fps, profile,
+          qmax = None, qmin = None):
```

Comment:
AFAICT, the hevc encoder in ffmpeg-qsv does not consider/handle qmin or qmax.  Please correct me if I'm wrong.

Reply:
Can we add  test feature in vaapi-fits , then to file a bug to developer ? I won't add reference for all test case.

Reply:
No.  If middleware does not support the feature, then a test for the feature should not be added.  Please consider the potential that there may be other users of this test framework.  And if we add the test, then those users could generate their own cases and baselines and erroneously assume they are correct.

If you believe the middleware should support this feature, please file a bug in the middleware.